main.tf

```diff
@@ -179,8 +292,12 @@ resource "google_container_cluster" "cluster" {
     }
   }
 
-  workload_identity_config {
-    workload_pool = "${var.project}.svc.id.goog"
+  dynamic "workload_identity_config" {
+    for_each = !var.enable_autopilot ? [1] : []
+
+    content {
+      workload_pool = "${local.project}.svc.id.goog"
+    }
```

Comment:
why does this depend on autopilot?

Reply:
because autopilot doesn't support `workload_identity_config`

Reply:
removed since we will move autopilot to a different module at some point